frontend/packages/container-security/src/plugin.ts

```diff
@@ -136,7 +139,7 @@ const plugin: Plugin<ConsumedExtensions> = [
     properties: {
       model: PodModel,
       page: {
-        name: 'Vulnerabilities',
+        name: i18next.t('details-page~Vulnerabilities'),
```

Comment:
Don't do it this way. Use the `%` syntax from #7234.

Reply:
Hi @spadgett, I udpated `frontend/packages/container-security/src/plugin.ts` to use the `%` for plugins:
```
     page: {
        // t('details-page~Vulnerabilities')
        name: '%details-page~Vulnerabilities%',
        href: 'vulnerabilities',
      },
```
However, in GUI the table col. header is rendering as "%details-page~Vulnerabilities%".
Looking at other plugin code I see the `%` used in plugin file of #7234, but don't see how it connects to `frontend/packages/console-plugin-sdk/src/utils/useTranslationExt.ts `? This plugin is in `frontend/packages/container-security/src/plugin.ts` so not sure if it uses `frontend/packages/console-plugin-sdk/src/utils/useTranslationExt.ts` internally?

Reply:
Removed plugin translation as it shouldn't be installed on CI cluster by default.